packages/ember-data/lib/system/model/model.js

```diff
@@ -611,7 +613,9 @@ var Model = Ember.Object.extend(Ember.Evented, {
   */
   changedAttributes: function() {
     var oldData = get(this._internalModel, '_data');
-    var newData = get(this._internalModel, '_attributes');
+    var currentData = get(this._internalModel, '_attributes');
```

Comment:
BTW, I wonder why here we have to use `get` instead of just `this._internalModel._whatever`